Fix tailwind conflict

mackenziequinnjetton opened the pull request:
Change config file extension
Fix syntax error in lockfile

mackenziequinnjetton committed:
Fix syntax error

mackenziequinnjetton committed:
Update package.json

clairenguyen committed:
Change order of plop and prettier in package-lock

clairenguyen committed:
Delete postcss.config.cjs

clairenguyen commented:
I added some small changes. Can someone else try loading it on their machines, confirm it works, and approve?

ksdenisova commented:
Do we still need these changes? I pulled up the main branch and I see the default home page.

mackenziequinnjetton commented:
> Do we still need these changes? I pulled up the main branch and I see the default home page.

We do, yes. Otherwise, the program throws an error.

karen-olson commented:
Looks good - it's working fine for me after fixing some issues I was having locally.

karen-olson reviewed (approved):
Looks good - it's working for me now. The issues I was having were just local.

clairenguyen committed:
Fetch and display restaurants  (no tests) (#30)

* Update testing dependencies
  - Downgrade @testing-library/jest-dom due to latest version having known issues with typescript types

* Fetch and display restaurant data on / restaurant route (no tests)

* fetch restaurants use client (#38)

* Update Restaurant type

---------

Co-authored-by: Brian Sung <6445410+seoulection@users.noreply.github.com>
